packages/contracts/test/cyberMarketplace.spec.ts

```diff
@@ -0,0 +1,210 @@
+import chai from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { ethers, network } from 'hardhat'
+import { expect } from 'chai'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { BigNumber, Contract } from 'ethers'
+
+import type {
+  DummyContract,
+  DummyNFT,
+  CyberMarketplace,
+} from '../typechain-types'
+
+const contracts: {
+  marketplace: CyberMarketplace
+  nft: DummyNFT
+  dummy: DummyContract
+} = {} as any
+
+const signers: Record<string, SignerWithAddress> = {}
+
+const KONG = 1
+const PUDGY = 2
+
+function days(n: number) {
+  return n * 24 * 60 * 60
+}
+
+function eth(n: number) {
+  return BigNumber.from(n).mul(BigNumber.from(10).pow(18))
+}
+
+function big(n: number) {
+  return BigNumber.from(n)
+}
+
+function delay(ms: number) {
```

Comment:
function not used?

Reply:
not anymore, was using it to test sell timeEnd, but settled on the more reliable `'evm_increaseTime'`

Reply:
Yeah do not relay on javascript time (when you can) because it depends on the speed computing of your machine, this kind of logic will failed on the overloaded CI of GitHub basically, evm_increaseTime is a much better way.